packages/core/src/browser/shell.ts

```diff
@@ -56,21 +64,85 @@ export interface DockLayoutData extends DockPanel.ILayoutConfig {
     activeWidgets?: Widget[]
 }
 
+export const MAINAREA_TABBAR_CONTEXT_MENU: string = 'mainarea-tabbar-context-menu';
```

Comment:
the proper type is `'mainarea-tabbar-context-menu'`, not string.

Reply:
OK